ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -173,10 +173,14 @@ def get_resources(dict_configs, cdump='gdas'):
         taskstr = f'{task.upper()}_{cdump.upper()}'
 
         strings.append(f'\t<!ENTITY QUEUE_{taskstr}     "{queuestr}">\n')
-        if scheduler in ['slurm'] and machine in ['ORION'] and task not in ['getic', 'arch']:
+        if scheduler in ['slurm'] and machine in ['ORION', 'JET'] and task not in ['getic', 'arch']:
             strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
         if scheduler in ['slurm'] and task in ['getic', 'arch']:
             strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_SERVICE;">\n')
+#        if scheduler in ['slurm'] and machine in ['JET'] and task in ['fcst']:
+#            strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
+#        if scheduler in ['slurm'] and machine in ['JET'] and task in ['post']:
+#            strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
```

Comment:
Please remove if not needed. Thanks!

Reply:
The fcst job failed during test with this change. I reverted the change and continue the test.

Reply:
Found the issue; modified it and testing continue with fcst job in the queue.